actors/builtin/reward/reward_actor.go

```diff
@@ -110,8 +110,8 @@ func (a Actor) computePerEpochReward(st *State, clockTime abi.ChainEpoch, networ
 	newSimpleSupply := big.Rsh(big.Mul(SimpleTotal, taylorSeriesExpansion(clockTime)), FixedPoint)
 	newBaselineSupply := big.Rsh(big.Mul(BaselineTotal, taylorSeriesExpansion(networkTime)), FixedPoint)
 
-	newSimpleMinted := big.Max(big.Sub(newSimpleSupply, st.SimpleSupply), big.Zero())
-	newBaselineMinted := big.Max(big.Sub(newBaselineSupply, st.BaselineSupply), big.Zero())
+	newSimpleMinted := big.Max(big.Sub(st.SimpleSupply, newSimpleSupply), big.Zero())
+	newBaselineMinted := big.Max(big.Sub(st.BaselineSupply, newBaselineSupply), big.Zero())
```

Comment:
This doesn't look right to me. If `newSimpleMinted` is positive, it would imply that at 116 St.SimpleSupply will be set to a smaller number than it used to be, meaning supply is contracting.

In interpret `st.SimpleSupply` as the total amount has has been minted so far, so should be increasing.

Reply:
I think that makes sense to me...

Though I think I've found the problem now

Reply:
newSimpleSupply should be greater than than st.SimpleSupply